lib/DDG/Spice/Meetup.pm

```diff
@@ -0,0 +1,62 @@
+package DDG::Spice::Meetup;
+# ABSTRACT: This Spice was inspired by an anonymous suggestion in the Instant Answers Forum
+# # # # # # source: https://duck.co/ideas/idea/1126/already-an-instant-answers-for-meetup
+# # # # # # When a user searches for "meetup + city name" they should receive
+# # # # # # instant answers of suggested meetups in that city sorted by the number of
+# # # # # # members in the group.
+
+use DDG::Spice;
+use YAML::XS 'LoadFile';
+use Text::Trim;
+
+spice is_cached => 1;
+
+name "Meetup";
+source "Meetup";
+icon_url "https://icons.duckduckgo.com/ip2/www.meetup.com.ico";
+description "Show meetups for a given city";
+primary_example_queries "meetups in philadelphia";
+secondary_example_queries "meetup groups near me";
+category "random";
+topics "social", "everyday", "travel";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Meetup.pm";
+attribution github => ["alohaas", "Olivia Haas"],
+            twitter => "livhaas";
+
+spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'https://api.meetup.com/2/groups?&key={{ENV{DDG_SPICE_MEETUP_APIKEY}}}&country=$2&city=$1&state=$3&radius=50.0&page=20&order=members&callback={{callback}}';
```

Comment:
It looks like it makes the wrong request when I'm searching outside of my city. So for example I want to see the meetups in San Francisco, I type in "meetups in san francisco", but it makes a request to this endpoint:

`http://window-roman.codio.io:5000/js/spice/meetup/san%20francisco/US/PA`

So it sends the right city (San Francisco), but it doesn't send the right state (PA).